src/main/java/bio/terra/service/dataset/flight/create/CreateDatasetRegisterIngestServiceAccountStep.java

```diff
@@ -22,7 +26,17 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
     String datasetServiceAccount =
         workingMap.get(DatasetWorkingMapKeys.SERVICE_ACCOUNT_EMAIL, String.class);
 
-    iamService.registerUser(datasetServiceAccount);
+    try {
+      iamService.registerUser(datasetServiceAccount);
+    } catch (Exception e) {
```

Comment:
Is there something one step down from Exception we can use? Like is there a superclass for `ApiException` or something? If there is I'd love to use that, cause this will also catch NPEs and such. If not, no worries and I'm okay moving forwards with this.